plugins/default/lib/test_nagios.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import unittest
+
+import nagios
+
+
+class TestParseResponse(unittest.TestCase):
```

Comment:
The name here is wrong, copy pasted from the other file :)

I don't see the point of testing constants.